### PR TITLE
Rename getRequest method for sf2.8 compatibility

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because it's a sf2.8 fix for PR #872.

## Changelog

```markdown
### Fixed
- Renamed internal method to fix sf2.8 incompatibility
```

## Subject

sf2.8 compatibility fix see [comment in PR #872](https://github.com/sonata-project/SonataPageBundle/pull/872#issuecomment-338478866)

### CODE REVIEW

File: Controller/PageController.php [hunk 1]
```diff
@@ -107,7 +107,7 @@ protected function getCmsManagerSelector()
     /**
      * @return Request
      */
-    private function getRequest()
+    private function getRequestObject()
```
Comment (greg0ire) [inline Controller/PageController.php]: I think the method itself could be greatly simplified. `request_stack` is guaranteed to exist now, isn't it?

Comment (acrobat) [reply Controller/PageController.php]: I  did this cleanup already in [PR #886](https://github.com/sonata-project/SonataPageBundle/pull/886/files#diff-004d0d5de4293bbc0343af54d205429eL112). But I can move that refactor to this PR if you like!

Comment (greg0ire) [reply Controller/PageController.php]: Oh no, it's perfect, don't bother!

File: Controller/PageController.php [hunk 2]
```diff
@@ -69,7 +69,7 @@ public function exceptionEditAction($code)
         $cms->setCurrentPage($page);
 
         // NEXT_MAJOR: remove the usage of $this->getRequest() by injecting the request action method (sf 2.4+)
-        return $this->getPageServiceManager()->execute($page, $this->getRequest());
```
Comment (OskarStark) [inline Controller/PageController.php]: please adjust the comment, too

```diff
- // NEXT_MAJOR: remove the usage of $this->getRequest() by injecting the request action method (sf 2.4+)
+ // NEXT_MAJOR: remove the usage of $this->getRequestObject() by injecting the request action method (sf 2.4+)
```

Comment (acrobat) [reply Controller/PageController.php]: done!

**Comment by acrobat** [comment]
Rebase done!

**Comment by core23** [comment]
Thanks @acrobat

**Pull request merged**